### PR TITLE
Bump angers to 0.23.0.dev decidim version 

### PR DESCRIPTION
**Pull request by MoretS**
Bump Angers to 0.23.0.dev decidim version

- Feature added : Max number of vote per project on decidim-budgets module
https://github.com/armandfardeau/decidim/tree/feature/maximum-number-of-vote-per-project

- Phone AH module : https://github.com/OpenSourcePolitics/decidim-module_phone_authorization_handler

### CODE REVIEW

File: config/initializers/decidim.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
   # Change these lines to set your preferred locales
   config.default_locale = :fr
-  config.available_locales = [:fr]
+  config.available_locales = [:en, :fr]
```
Comment (armandfardeau) [inline config/initializers/decidim.rb:17]: Does Angers is supposed to be in English too?

File: db/migrate/20190717132653_fix_user_names.decidim.rb [hunk 2]
```diff
@@ -17,8 +17,7 @@ def change
 
         entity.name = entity.name.delete(characters_to_remove).strip
         sanitized_nickname = entity.nickname.delete(characters_to_remove).strip
-        entity.nickname = Decidim::UserBaseEntity.nicknamize(sanitized_nickname, organization: entity.organization)
-        entity.save!
+        entity.update_columns(nickname: Decidim::UserBaseEntity.nicknamize(sanitized_nickname, organization: entity.organization))
```
Comment (armandfardeau) [inline db/migrate/20190717132653_fix_user_names.decidim.rb:20]:
```suggestion
        entity.update!(nickname: Decidim::UserBaseEntity.nicknamize(sanitized_nickname, organization: entity.organization))
```

File: db/migrate/20200203145836_add_custom_link_to_decidim_conferences.decidim_conferences.rb [hunk 3]
```diff
@@ -1,10 +0,0 @@
-# frozen_string_literal: true
-# This migration comes from decidim_conferences (originally 20191003080307)
-
-class AddCustomLinkToDecidimConferences < ActiveRecord::Migration[5.2]
-  def change
-    add_column :decidim_conferences, :custom_link_name, :jsonb
-    add_column :decidim_conferences, :custom_link_url, :string
-    add_column :decidim_conferences, :custom_link_enabled, :boolean, default: false
-  end
-end
```
Comment (armandfardeau) [inline db/migrate/20200203145836_add_custom_link_to_decidim_conferences.decidim_conferences.rb:10]: You removed the migration, but the DecidimConferences tables are still in the database, is it okay?

**Comment by armandfardeau** [comment]
@MoretS See my comments above and wait a second I need to test it.

**Comment by armandfardeau** [comment]
@MoretS Tested and good to go as soon as you answered my question

File: db/migrate/20200203145836_add_custom_link_to_decidim_conferences.decidim_conferences.rb (hunk 3 above)
Comment (MoretS) [reply db/migrate/20200203145836_add_custom_link_to_decidim_conferences.decidim_conferences.rb:10]: @armandfardeau I removed it due to an error when I tried to db:migrate but I can't really remember what the error was (except that I'm pretty sure that it was not a duplicate migration error). 

Actually, the running version of Angers does not have this migration and no errors were reported.

Should I try to add it again and test to run migrations? 

Thanks!

**Pull request merged**